### PR TITLE
Zijie lint fix reducer phase 1

### PR DESCRIPTION
**Pull request by iTvX**
# Description
This is lint fix for the actionltemsReducer.js, allProjectsReducer.js, allTasksReducer.js, allTeamsReducer.js, allTimeEntriesReducer.js from src/reducers file

CleanShot 2023-10-18 at 21 41 06@2x

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user

(Please be sure to record a video)
5. **Verify that the project/task/teams/time entries reducer is working normally.**

~~6. Delete .eslintignore file in the root: `rm .eslintignore`
7. `npx eslint src/reducers/actionItemsReducer.js`
8. `npx eslint src/reducers/allProjectsReducer.js`
9. `npx eslint src/reducers/allTasksReducer.js`
10. `npx eslint src/reducers/allTeamsReducer.js`
11. `npx eslint src/reducers/allTimeEntriesReducer.js`
12. `git restore .`~~

### CODE REVIEW

**Comment by olenadanykh** [comment]
Hi @iTvX. I've tested your PR, and all the functionality remains unchanged. I also didn't notice any ESLint errors. Great work!

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49422649/9c3ac58f-25c3-4c72-84ee-7065a7d8534b

**Comment by Iris-Song** [comment]
hi，I've tested your PR, and all the functionality in projects and teams remains unchanged. Good job
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/58033867/449c894d-2d5d-4692-a2f4-314577413bdd

**Review by Iris-Song — APPROVED** [review]
hi，I've tested your PR, and all the functionality in projects and teams remains unchanged. Good job
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/58033867/449c894d-2d5d-4692-a2f4-314577413bdd

**Review by StrawberryCalpico — COMMENTED** [review]
Hi, I tested your PR, the project/task/teams/time works normally. 
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114443664/72256160-c6a0-45bb-acf2-6f9549788097
No error
Screenshot 2023-10-20 at 11 29 04 PM

**Review by JYXiao-2021 — APPROVED** [review]
Hi, I test this and it works as intended.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49748716/333c60bc-d89e-42b7-90c0-4bc7f30099df

Screenshot 2023-10-21 at 12 18 12 AM

**Review by Shubhu0 — APPROVED** [review]
HI @iTvX Reviewed and Tested the PR and it is working as expected! No issues with lint as well


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/56580102/c6b9e857-e45e-4e98-abcf-4eaa7d64fa46

image

**Comment by RheaWu1212** [comment]
Hi the PR working as expected!

Screen Shot 2023-10-21 at 11 49 19

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/109626741/70077b1d-290e-4958-ace6-a054212bd358

**Review by TheDescendant39 — APPROVED** [review]
Hi @iTvX, I checked all the files as listed and checked common features to test if the app was running well. 

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/47534216/bf6b255d-35c8-4a39-9e3c-90e8f1b92d48
Screen Shot 2023-10-21 at 3 07 19 PM

**Review by malikjahanzaib — APPROVED** [review]
Hi @iTvX, I have tested this PR locally, no lint errors and rest works normally. Checkout my screenshot and test recording below.
Screenshot:
Screenshot 2023-10-21 154951

Test rec:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46221334/607e0ba7-7a62-4cfd-9dc4-f901d73883b3

**Comment by palakgosalia** [comment]
Hey, I have reviewed the PR and tested it. Did not get any lint issues and everything works well. Thank you

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/104417272/4511eee9-888e-44cf-8b71-59bddd34bdfa

**Review by DavidC0126 — APPROVED** [review]
Hi @iTvX I tested the PR and it works as intended. No error!


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/74082492/cf387835-c354-4a95-be63-9ee9c94894aa

eslint

**Comment by kavya151** [comment]
I have verified projects,teams and timelog are working normal and I didn't get any lint issues.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/61841929/c19a0461-5c33-4ab4-bd00-a021ce4cb34e

File: src/reducers/actionItemsReducer.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export,default-param-last
```
Comment (EvianTan) [inline src/reducers/actionItemsReducer.js:1]: Please fix this error, not using "eslint-disable-next-line"

File: src/reducers/allProjectsReducer.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ export const updateObject = (oldObject, updatedProperties) => {
   };
 };
 
+// eslint-disable-next-line default-param-last
```
Comment (EvianTan) [inline src/reducers/allProjectsReducer.js:17]: Same as above.

File: src/reducers/allProjectsReducer.js [hunk 3]
```diff
@@ -43,7 +44,8 @@ export const allProjectsReducer = (allProjects = allProjectsInital, action) => {
           ]),
         });
       }
-    case types.UPDATE_PROJECT:
+    // eslint-disable-next-line no-fallthrough
```
Comment (EvianTan) [inline src/reducers/allProjectsReducer.js:47]: Same as above.

File: src/reducers/allTasksReducer.js [hunk 4]
```diff
@@ -46,6 +45,7 @@ const sortByNum = tasks => {
   return filterAndSort(appendTasks, 4);
 };
 
+// eslint-disable-next-line import/prefer-default-export,default-param-last
```
Comment (EvianTan) [inline src/reducers/allTasksReducer.js:48]: Same as above.

File: src/reducers/allTasksReducer.js [hunk 5]
```diff
@@ -54,6 +54,7 @@ export const taskReducer = (allTasks = allTasksInital, action) => {
     case types.FETCH_TASKS_ERROR:
       return { ...allTasks, fetched: true, fetching: false, error: action.err };
     case types.RECEIVE_TASKS:
+      // eslint-disable-next-line no-param-reassign
```
Comment (EvianTan) [inline src/reducers/allTasksReducer.js:57]: Same as above.

File: src/reducers/allTeamsReducer.js [hunk 6]
```diff
@@ -12,6 +12,7 @@ export const updateObject = (oldObject, updatedProperties) => ({
   ...updatedProperties,
 });
 
+// eslint-disable-next-line default-param-last
```
Comment (EvianTan) [inline src/reducers/allTeamsReducer.js:15]: Same as above.

File: src/reducers/allTimeEntriesReducer.js [hunk 7]
```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export,default-param-last
```
Comment (EvianTan) [inline src/reducers/allTimeEntriesReducer.js:1]: Same as above.

**Pull request closed without merging**